test/lib/elven_gard/helpers/packet_test.exs

```diff
@@ -206,10 +206,10 @@ defmodule ElvenGard.Helpers.PacketTest do
     end
 
     test "documentations previously set" do
-      got = HandlersE.elven_get_packet_documentation()
+      got = HandlersE.get_packet_definitions()
```

Comment:
What do `E` mean ?

Reply:
Nothing, I just incremented a letter for all handlers

Reply:
You should at least name it based on what the mock is suppose be used for. For example `MultiplePacketHandler` or `BadValueHandler`...

Reply:
You're right, I'll change it